lib/accumulate_distribute/events/life_start.js

```diff
@@ -32,7 +32,7 @@ const onLifeStart = async (instance = {}) => {
   )
 
   if (hasIndicatorOffset(args)) {
-    const IndicatorClass = HFI[relativeOffset.type.toUpperCase()]
+    const IndicatorClass = HFI[relativeOffset.type === 'ma' ? 'SMA' : 'EMA']
```

Comment:
is it possible to index HFI by the final type instead of doing this check?

Reply:
need to check for its dependency through out the algo .. will change it.. thanks!!

Reply:
Yes.. closing